Evolutionary and ecological forces that influence among population diversification can be classified as either neutral and putatively adaptive (though see Hahn 2010).  The spatial distribution of neutral genetic variation is the primary source of data used to reconstruct patterns of gene flow and population connectivity (Dyer et al. 2010; 2012), reconstruct historical phylogeography and test phylogeographic hypotheses (Eckert et al. 2008; Garrick et al. 2009), identify spatial patterns of parentage (Nason et al. 1998), estimate relatedness (Queller & Goodnight 1989), and identify transgenic gene escape (Stewart et al. 2003).  Putatively adaptive genetic variance is often identified by means of comparison to neutral structure by looking for signals such as selective sweeps (Pollinger et al. 2005; Voight et al. 2006; Sella et al. 2009), loci with increased population structure (Coop et al. 2009), and correlations between allelic variation and bioclimatic factors (Coop et al. 2010; Eckert et al. 2010a).  Adaptive variance is constrained, historically, by neutral demographic history resulting in spatial genetic signatures even with genomic regions experiencing strong selective pressure (Nielsen 2005, Coop et al. 2010).  Teasing apart the confounding signals of selection with linked variants (e.g., Maynard Smith and Haigh 1974; Hermisson and Pennings 2005; Stephen et al. 2006; McVean 2007) from genome-wide patterns due to neutral history is an ongoing methodological challenge, a task that is becoming increasingly difficult with the increasing size of genomic datasets (Stapley et al. 2010, Barrett and Hoekstra 2011).

A common approach to identify genomic regions carrying signatures of selection is to estimate parameters such as nucleotide diversity (π or some function of nucleotide diversity such as Tajima’s D; Nielsen et al. 2005), linkage disequilibrium (Nielsen 2005), or population divergence (FST; Beaumont & Balding 2004; Foll & Gaggiotti 2008) and compare them to expectations based upon neutral theory and a priori demographic models (e.g., Chen et al. 2012).  Natural selection affects magnitudes of these parameters, as for example the larger values of FST, increased levels of linkage disequilibrium, and decreased values of nucleotide diversity expected at regions experiencing selective sweeps due to directional selection. While an approach based on a single parameter provides evidence for the presence of selection localized within genomes, it provides few insights into how selection operated differentially among populations and may suffer from problems of identifiability of causative processes.  

Here we introduce the use of an additional approach, rather than estimating individual statistics to identify the presence of selection, we propose the use of conditional covariance networks (e.g., Population Graphs) that will characterize how selection has influenced the pattern of among population genetic covariance.  Changing the focus from the magnitude of a single statistic like FST to the topological shape of a covariance network also provides opportunities for subsequent categorization of non-neutral genomic regions and an a priori hypothesis testing framework.  We begin with a brief review of conditional genetic covariance and how it is used in Population Graphs.  Simulations are used to identify a set of graph-theoretic parameters that change under the operation of selection (both positive and negative) differentiating from those expected due to neutral processes and some forms of demography.  This approach was applied to an example dataset, the HGDP-CEPH dataset for chromosome 2, for which we have numerous examples of positive selection in the global human population.
