
### Population Graphs

A Population Graph (popgraph hereafter from Dyer & Nason 2004, Dyer 2015) is a mathematical graph, $G$, defined by conditional genetic covariance (Whittaker 1990), whose purpose is to describe the spatial distribution of genetic covariance among sampled locales.  Each popgraph is constructed solely from population genetic data and consist of two components, nodes and edges.  The node set, $V=\{v_1,v_2, \ldots, v_k\}$, represent sampled populations, each of which has a size associated with it that is scaled to the within population multilocus genetic variance, $\sigma_w^2$.  Pairs of nodes may be connected by an edge set, $E=\{e_1,e_2,\ldots ,e_m\}$, whose presence is determined by conditional genetic covariance (Dyer 2015).  Edge lengths are weighted relative to the contribution the pair of populations contribute to the overall estimate of among strata genetic covariance as
\[
\sigma_A^2=\sum_{i\ne j}^K \sigma_{A; i \leftrightarrow j}^2
\].

This decomposition is fundamentally different in that what we are doing is partitioning the total among strata genetic covariance and not the sample covariance based upon allele frequencies across populations (Dyer & Nason 2004).


{%Transition into conditionality here %} Two populations are said to be conditionally independent if and only if they have zero covariance after conditioning their individual variances on the rest of the dataset.  As defined,  nodes $v_i$ and $v_j$ are conditionally independent of each other given the rest of the nodes $v_{k;\;\forall k‚â†i,j}$ if and only if:

\[
v_i ‚üÇ v_j | v_k = P(v_i ‚à© v_j | v_k) = P(v_i|v_k)P(v_j|v_k)
\]


Nodes found to be conditionally independent in a Population Graph not connected, even though the unconditional covariance between them may deviate from zero. 

Biologically, conditional genetic independence can arise under several scenarios. For example, consider a source-sink demographic model.  Divergence among sink populations will be small (e.g., they will have high covariance) even if there is no inter-sink gene flow because they all receive migrants from the same source population.  Conditionally though, sink populations will be genetically independent, conditional on how they covary with the source population.  The net result of conditional covariance is that it more closely resolves the underlying demographic model than pairwise approaches that only consider a subset of the data.  The Population Graph G={V,E} is a also an independence graph with pairwise, local, and global Markov properties.  This means that populations not directly connected within the topology are statistically independent conditional on how they covary with the rest of the nodes in the network.  Changing the node set by adding or removing a population modifies the entire covariance structure, since all edges are based upon the totality of the network (cf., Koen et al. 2013). 

The fundamental characteristic of a Population Graph is the underlying conditional covariance topology, created in large part by the historical demographic history of populations (Lowry et al. 2008, Baker & Dyer 2011, Garroway et al. 2011, Dyer et al. 2012, Traor√© et al. 2012, Winkler et al. 2012).  All other analyses are based upon either the magnitude of structure or the amount of diversity relative to a null model.  This changes the focus of the analysis from interpreting a set of indices to analyzing the multivariate shape of population covariance.  For example, different evolutionary histories can lead to indistinguishable FST estimates, whereas the topologies evolved in these scenarios create divergent topological shapes (Figure X-Interpretive, three panel, same Fst different shapes).  

Conditional genetic covariance as implemented in Population Graphs also experience a more rapid trajectory towards variance stabilization when compared to pairwise parameters (Dyer 2007) making it more responsive to changes and can provide increased resolution (e.g., a net increase in R2 in isolation by distance scenarios) in describing the spatial distribution of genetic variation (Dyer et al. 2010).  A Population Graph topology is defined as a ‚Äòmodel-free‚Äô analysis of genetic covariance even though certain structural features of the network are correlated with common population genetic parameters (Dyer 2007).  For example, node connectivity is correlated with genetic structure (e.g., FST ùúå = -0.95) and the diameter of the graph is related to the regression coefficient for analysis of isolation by distance (ùúå = -0.98).  The derived topology of genetic covariance can be converted into a measure of genetic distance, called cGD (Dyer et al. 2010), which is both more sensitive to demographic changes and provides a higher level of precision than normal pairwise structure and distance parameters (Dyer et al. 2010).  As with all genetic distances, cGD is represented as a matrix of pairwise distances, however cGD is derived from the topology of the conditional covariance graph rather than isolated pairs of strata.

Adopting a topological analysis provides a distinctly different focus than that based upon estimation and interpretation of individual parameters.  At a fundamental level, individual estimators of structure and diversity, such as FST and œÄ, are defined based upon an overall magnitude, that is compared to estimates from other regions of the genome.  Divergence in excess of the background level is one way to identify the footprints of selection, though it is not the only way.  A Population Graph topology says relatively little about the magnitude of variation but rather describes how that variation is distributed across populations.  

It is precisely the way in which the topological shape changes as we walk down a chromosome that allows us to identify genomic regions between neutral and selective generating processes and to gain inferences about the ways in which alternative selection processes have shaped different genomic regions.  

¬∂ Paragraph on topological classification & comparisons (Box for measurements)

## Simulations

Adopting a simulation approach, we expand the analyses from Dyer (2007) to determine which sets of graph-theoretic features within a Population Graph topology are sensitive to the action of selection (both positive and negative).  From simulated data we quantify the effects of selection on parameters describing features of nodes, edges, and the overall topological shape.  Node parameters include degree (the number of edges connected to a node), node betweenness (the number of shortest paths through the entire topology that go through this node), node closeness (the distance from a node to all others in the topology), and cGD (a measure of pairwise conditional distance based on shortest paths).  For edges, we also edge betweenness (an analogous measure quantifying the relative number of shortest paths that traverse a particular edge).  Changes in overall graph structure are estimated using the slope degree (a measure of the degree distribution that differentiates scale free graphs from random networks) and graph diameter (the length of the path connecting the most distant nodes in a network).  These parameters will serve as a multivariate description of topological structure.

To determine if selection the way in which a Population Graph topology is constructed, we used forward time simulations under a Wright-Fisher model with demography and natural selection.  A set of twenty-five populations were defined with Œò = 4Ne¬µ = 5.0 arranged in a one-dimensional stepping-stone configuration with symmetric migration between neighboring populations at a rate equal to Nem = 9.5.  Simulations were iterated to allow burn-in for 50Ne generations after which 25 individuals were randomly selected and 100 unlinked SNP loci were retained.  This process was replicated 100 times under three different evolutionary scenarios for a subset of loci: neutrality (hereafter NEU), negative (purifying) selection (NEG), and positive (directional) selection (POS).  The non-neutral replicates (NEG & POS) were simulated such that a fraction of the new mutations (we used 70%) in 20% of the populations were either deleterious or advantageous (2Nes = 50).  In the remaining 80% of the populations, these loci were neutral. Thus, our simulations are essentially those of a model of local adaptation via conditional neutrality, which is a difficult case to detect using traditional population genetic methods (Tiffin & Ross-Ibarra, 2014).  We desired the majority of populations to evolve neutrality and for the remaining to have moderately strong magnitudes of selection in order to quantify the performance of this method with obvious signals of selection. The parameters of the underlying migration model and selection regimes were chosen for generality and were not intended to mimic specific life histories, though they are consistent with other simulation approaches applied in quantifying model performance.  All simulations were performed using the Selection on Finite Sites under COmplex Demographic Events (SFS_CODE) from Hernandez (2008).  At the end of each simulation, multilocus genotypes were used to estimate a Population Graph with the gstudio library (http://github.com/dyerlab/gstudio) in R (R Core Team 2016).  Graph-theoretic parameters were extracted from each topology using functions from the igraph (Gabor & Nepusz 2006) and popgraph (http://github.com/dyerlab/popgraph) libraries in R.  Fixed effects ANOVA and TukeyHSD post-hoc tests were used to identify significant differences of graph-theoretic parameters among treatments of NEU, NEG, and POS selection regimes using R. 

## HGDP-CEPH Chromosome 2

To demonstrate the utility of Chromosome Walking we used the Human Genome Diversity Project (HGDP) - CEPH SNP dataset for chromosome 2. Due to the highly complex evolutionary history associated with the rather recent emergence of our species, patterns of demography and natural selection are challenging to tease apart, especially with respect to bouts of recent directional selection and local adaptation (Tishkoff and Verrelli 2003a; Tishkoff and Verrelli 2003b). Although whole genome datasets are emerging for select samples, few available datasets compared to the HGDP-CEPH dataset that includes over 1 million SNP loci genotyped in ~1,050 unrelated individuals sampled from 52 ethnically and geographically diverse populations of Africa, Europe, the Middle East, South and Central Asia, East Asia, Oceania and the Americas. This dataset has been an extensive resource in characterizing human population ancestry (e.g., Cann et al. 2002; Rosenberg et al. 2002), as well as identifying signatures of natural selection at the genome and single gene levels (e.g., Sabeti et al. 2007), from a global perspective. Several strong candidates for positive selection have been revealed on chromosome 2 (Voight et al. 2006; Pickrell et al. 2009), and in fact, one of the more championed signatures of recent directional selection in this dataset is found at the lactase (LCT) gene (Bersaglieri et al. 2004), as evidenced by significantly high FST and unusually long haplotype blocks associated with lactase persistence alleles. Thus, as an initial ‚Äúproof-of-concept‚Äù example, the chromosome 2 dataset, which includes 42,588 genotyped SNPs in the HGDP-CEPH panel, represents a perfect candidate with which to apply our approach, especially given the opportunity to include the LCT gene in our analyses. 

We analyzed among-population covariance on these data by estimating Population Graph topologies sequentially (Figure 1) along chromosome 2, using partially overlapping sets of SNP loci. The size of the sampling window is dependent upon SNP density and overall genetic diversity.  More loci are always preferable, though their cumulative contribution is highly asymptotic with respect to the shape of the resulting topology.  Increasing the number of loci used to estimate a single topology also results in a larger window along the chromosome, which we are interested in minimizing.  To find an optimal window size for this example data set, we subsampled contiguous stretches of SNPs in sets of 10, 20, 30, 40, 50, 60, 70, and 80 loci at random starting locations from the set of all 42,588 available SNPs.  For each window size, we sampled 100 topologies.  We selected the smallest window size whose variance in number of edges in the resulting topologies had stabilized. 
rphological) to scan genomes for congruent covariance structure due to selection.  We close by discussing the utility and limitations of using a Population Graph approach in characterizing non-neutral genetic covariance.
